src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -209,20 +209,20 @@ class OperatorStore extends BaseStore {
   /**
    * Get max validators count
    */
-  async getOperatorValidatorsLimit(): Promise<number> {
+  async updateOperatorValidatorsLimit(): Promise<void> {
     const walletStore: WalletStore = this.getStore('Wallet');
     const contract: Contract = walletStore.getterContract;
     if (this.operatorValidatorsLimit === 0) {
       this.operatorValidatorsLimit = await contract.methods.getValidatorsPerOperatorLimit().call();
     }
-    return this.operatorValidatorsLimit;
   }
 
   /**
    * Check if operator registrable
    */
-  isOperatorRegistrable(validatorsRegisteredCount: number) {
-    return this.operatorValidatorsLimit > validatorsRegisteredCount;
+  isOperatorRegistrable(validatorsRegisteredCount: number): boolean {
```

Comment:
typo, should be `isOperatorRegisterable`. But better name in common would be `operatorReachedValidatorsLimit`

Reply:
renamed